substrate/utils/wasm-builder/src/wasm_project.rs

```diff
@@ -855,7 +855,7 @@ fn build_bloaty_blob(
 	println!("{} {}", colorize_info_message("Using rustc version:"), cargo_cmd.rustc_version());
 
 	// Use `process::exit(1)` to have a clean error output.
-	if build_cmd.status().map(|s| s.success()).is_err() {
+	if !matches!(build_cmd.status().map(|s| s.success()), Ok(true)) {
```

Comment:
```suggestion
	if build_cmd.status().map_or(true, |s| !s.success()) {
```

Reply:
Sorry, I added this to the merge queue before I saw your comment. I can change this in my next PR.

Reply:
Not that important :)